src/components/Dropdown/Dropdown.tsx

```diff
@@ -51,10 +54,12 @@ interface State {
  * */
 export class Dropdown extends React.Component<DropdownProps, State> {
   static displayName = 'Dropdown';
+  static contextType = TPAComponentsContext;
   static defaultProps: DefaultProps = {
     placeholder: '',
     options: [],
     placement: 'bottom',
+    shouldRenderNativeSelectOnMobile: false,
```

Comment:
Let's change this to be `true` by default.
This prop will give consumers a backdoor

Reply:
And maybe let's change it to just `mobileNativeSelect` 😬 ?

Reply:
that is definitely a better name. I'll change it.